### PR TITLE
EDI: Some fixes

### PR DESCRIPTION
**Pull request by etobella**
This PR adds two changes:

- Fixes the fields_view_get, as it was loosing the specific views on some fields (for example, on invoice lines)
- Adds a new function to search a exchange record (similar to _has_exchange_record, but returning the record)

### CODE REVIEW

File: edi/models/edi_exchange_consumer_mixin.py [hunk 1]
```diff
@@ -139,6 +142,17 @@ def edi_create_exchange_record(self, exchange_type_id):
 
     def _has_exchange_record(self, exchange_type, backend=False, extra_domain=False):
         """This function is useful when generating the configuration"""
```
Comment (simahawk) [inline edi/models/edi_exchange_consumer_mixin.py]: can you change the docstrings as you are here? :wink:

**Review by simahawk — APPROVED** [review]
LG, small remark

File: edi/models/edi_exchange_consumer_mixin.py (hunk 1 above)
Comment (etobella) [reply edi/models/edi_exchange_consumer_mixin.py]: Done

**Review by jarroyomorales — APPROVED** [review]
LGTM :+1:

**Pull request merged**